docs/Shipment.java

```diff
@@ -0,0 +1,53 @@
+public class Shipment(){
+	
+	String shipmentId; 
+	String shipmentMethod;	
+	String currentWarehouseId;
+	float shipmentWeight;
+	long receivedAt;
+	
+	//No argument constructor not provided
+	public Shipment(String shipId, String shipMthd, float weight){	
+		shipmentId = shipID;
+		shipmentMethod = shipMthd;
+		shipmentWeight = weight;
+	}
+	
+	public Shipment(String shipId, String shipMthd, String warehouse, float weight, long received){
+	
+		shipmentId = shipID;
+		shipmentMethod = shipMthd;
+		currentWarehouseId = 
+		shipmentWeight = weight;
+		receivedAt = received;		
+	}
+
+	//Basic getters and setters
+	//No setters for shipmentID, shipmentMethod and warehouseID
```

Comment:
This comment could be clarified - it says there isn't a setter for the warehouseId but then starts off with the warehouse ID setter.